### DISCUSSION TITLE
error at simulation time

### DISCUSSION THREAD

**Discussion by matheusposilva** [OpenFAST/openfast#1281] [General]
I'm trying to run Openfast with the Spar example, but when I change the time simulation to more than 60s the program shows this problem. Can anyone help me with this

![image](https://user-images.githubusercontent.com/108477246/194902617-dfb279d1-9527-46b9-a6ae-18f6e91f8118.png)

**Comment by jjonkman** [comment]
Dear @matheusposilva,

Similar questions have been asked on our forum, e.g., here https://forums.nrel.gov/t/error-ff-wind-array-was-exhausted/1267.

The OpenFAST r-test only provides sample input; the wind files, in particular, were purposely made short to reduce the size of the r-test archive. You should run TurbSim to generate the turbulent wind data that you need for your own use--longer simulations require a longer time series of wind inflow.

Best regards,
